rtl/src/gtest/calls.rs

```diff
@@ -73,6 +78,20 @@ impl GTestRemoting {
         Ok(reply.payload().to_vec())
     }
 
+    fn extract_events(self, run_result: &RunResult) {
+        let events: Vec<(ActorId, MessageId, Vec<u8>)> = run_result
+            .log()
+            .iter()
+            .filter(|entry| entry.destination() == ActorId::zero())
+            .map(|entry| (entry.source(), entry.id(), entry.payload().to_vec()))
+            .collect();
+        for listener in self.listeners.borrow_mut().iter_mut() {
+            if let Some(listener) = listener.upgrade() {
+                listener.events.borrow_mut().append(&mut events.clone());
+            }
```

Comment:
Do you want to drop dead weak refs from the vec?

Reply:
non neccesary, couse it all for tests only

Reply:
added containers.retain berfore append new events